addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -1347,6 +1349,15 @@ var Wysiwyg = Widget.extend({
         }
         this.$attribute_translations = $('.o_editable_translatable_attribute');
         this.translations = [];
+        // (id -> value) map of the initial values of a translation, to be able to determine
+        // whether a translation has been modified or not, so as to not save unmodified translation
+        // (important because when a translation is saved, Odoo marks it as translated)
+        const translationsInitialValues = $('[data-oe-translation-id]').toArray().reduce((acc, elem) => {
+            acc[elem.getAttribute('data-oe-translation-id')] = elem.textContent
+            return acc;
+        }, {});
+        // checks whether the translation has been modifier or not
+        this.isTranslationPristine = elem => translationsInitialValues[elem.getAttribute('data-oe-translation-id')] === elem.textContent;
```

Comment:
Which would you create the method inside this method ?
All the other method are being defined directly in the class, I would do use the standard convention.

Reply:
It allows me to encapsulate the map in the closure, I felt it was cleaner since no one else is supposed to access it, and the function is not gonna work until the map is initialized anyway.

Reply:
:+1: